app/concepts/api/v1/users/lib/service/redis_adapter.rb

```diff
@@ -27,7 +27,9 @@ def initialize(token)
     end
 
     def push
-      storage.setex(token_name, @token_exp, @token)
+      storage.setex(token_name, @token_exp.to_i, @token)
+    rescue Redis::CommandError => e
```

Comment:
What is the problem was?

I'm not sure but looks like a problem related to the passed parameters